java/rocksjni/portal.h

```diff
@@ -607,7 +607,7 @@ class WriteTypeJni {
     }
 
     // Get the DELETE enum field of org.rocksdb.WBWIRocksIterator.WriteType
-    static jobject DELETE(JNIEnv* env) {
+    static jobject DELETE_(JNIEnv* env) {
```

Comment:
I think that might be better, otherwise we do not have a consistent naming for these enumerations.